x-pack/plugins/lens/public/indexpattern_datasource/datapanel.test.tsx

```diff
@@ -856,24 +860,26 @@ describe('IndexPattern Data Panel', () => {
         );
         const wrapper = mountWithIntl(<InnerIndexPatternDataPanel {...props} />);
         act(() => {
-          (
-            wrapper
-              .find('[data-test-subj="lnsIndexPatternActions-popover"]')
-              .first()
-              .prop('children') as ReactElement
-          ).props.items[0].props.onClick();
```

Comment:
Thanks to mocking the portal: `ReactDOM.createPortal = jest.fn((element) => element)` we can just assume that the content of the popover will be placed directly inside the popover. And thanks to that we can avoid the ugly lines above.